diff --git a/CMakeLists.txt b/CMakeLists.txt
index e07a489..180eb6c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,13 +1,13 @@
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 2.8)
 project(gamgee)
 
 include(ExternalProject)
 
 # This is a C++14 library
-add_compile_options("-std=c++1y")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++1y")
 
 # Dependency: Boost Unit Test Framework (find in the system)
-find_package(Boost 1.55 COMPONENTS unit_test_framework REQUIRED)
+find_package(Boost 1.46 COMPONENTS unit_test_framework REQUIRED)
 include_directories(${Boost_INCLUDE_DIRS})
 
 # enable installing dependencies
@@ -21,7 +21,6 @@ include("contrib/htslib.cmake")
 include_directories(gamgee)
 
 add_subdirectory(gamgee)
-add_subdirectory(test)
 
 ADD_CUSTOM_TARGET(debug
   COMMAND ${CMAKE_COMMAND} -DCMAKE_BUILD_TYPE=Debug ${CMAKE_SOURCE_DIR}
